### PR TITLE
chore: remove jquery dep

### PR DESCRIPTION
**Pull request by rbarilani**
* removing "jquery" dep fix CVE issues:
    - https://nvd.nist.gov/vuln/detail/CVE-2015-9251 should not be so serious anymore.
    - Future updates regarding https://nvd.nist.gov/vuln/detail/CVE-2016-10707 might be needed.

* fix gulp-sourcemaps version to 1.6.0, updating was causing error:
  ```
  Error: Cannot find module 'debug'
  ```

* add package-lock.json file introduced by new versions of `npm`

### CODE REVIEW

**Comment by rbarilani** [comment]
👍

File: package.json [hunk 1]
```diff
@@ -50,7 +50,6 @@
     "flux": "^2.1.1",
     "imagemin-pngquant": "^4.2.2",
     "jest-cli": "^11.0.2",
-    "jquery": "^2.2.1",
```
Comment (fokusferit) [inline package.json:53]: +1

**Comment by fokusferit** [comment]
@rbarilani I'm a bit unsure regarding package-lock commits. Even with that npm builds might fail, the package-lock changes from time to time heavily (30K LOC ^^). I hesitate a bit here.

**Comment by rbarilani** [comment]
@fokusferit Yeah I'm also a bit skeptical about npm@5 and the package-lock file but I'm trying to follow what NPM suggests... put the package-lock.json under version control, in alternative I can add the package-lock to .gitignore... WDYT?

**Comment by fokusferit** [comment]
👍

**Comment by fokusferit** [comment]
I've merged it. Let see what experience we will gain from that :) (if any). I checked also the npm docs and you're right, it is recommended.

**Pull request merged**